pkg/scheduler/framework/runtime/framework.go

```diff
@@ -53,6 +53,9 @@ func (frw *frameworkImpl) RunFilterPlugins(ctx context.Context, placement *polic
 	for _, p := range frw.filterPlugins {
 		pluginResult := p.Filter(ctx, placement, resource, cluster)
 		result[p.Name()] = pluginResult
```

Comment:
resultmap just save the result about part of filterPlugin. it is better to return the failure result

Reply:
Failure reasons are already recorded in `Filter` result, do I understood your words correctly?

Reply:
I think it is better to just record the failure reasons and remove the success record if it would return once one plugin filtered the cluster

Reply:
Yes, it was updated as you advised. Now, I will only return a `*Result` instead of a `Map` struct. PTAL thanks a lot.